frontend/packages/kubevirt-plugin/src/components/modals/clone-vm-modal/configuration-summary.tsx

```diff
@@ -0,0 +1,118 @@
+import * as React from 'react';
+import { DASH, getName, getNamespace } from '@console/shared';
+import { K8sResourceKind } from '@console/internal/module/k8s';
+import { CUSTOM_FLAVOR } from '../../../constants/vm';
+import { VMKind } from '../../../types/vm';
+import {
+  getDataVolumeTemplates,
+  getDisks,
+  getFlavor,
+  getFlavorDescription,
+  getInterfaces,
+  getOperatingSystemName,
+  getVolumes,
+  getWorkloadProfile,
+} from '../../../selectors/vm';
+import { getStorageSize } from '../../../selectors/selectors';
+import {
+  getDataVolumeResources,
+  getDataVolumeStorageClassName,
+} from '../../../selectors/dv/selectors';
+import { getPvcResources, getPvcStorageClassName } from '../../../selectors/pvc/selectors';
+
+import './_clone-vm-modal.scss';
+
+const getNicsDescription = (vm: VMKind) => {
+  const interfaces = getInterfaces(vm);
+  return interfaces
+    .map((intface) => [intface.name, intface.model, intface.macAddress].filter((i) => !!i))
+    .map((intface) => <div key={intface[0]}>{intface.join(' - ')}</div>);
+};
+
+const getDisksDescription = (
+  vm: VMKind,
+  pvcs: K8sResourceKind[],
+  dataVolumes: K8sResourceKind[],
+) => {
+  const disks = getDisks(vm);
+  const volumes = getVolumes(vm);
+  const dataVolumeTemplates = getDataVolumeTemplates(vm);
+  return disks.map((disk) => {
+    const description = [];
+    description.push(disk.name);
+
+    const volume = volumes.find((v) => v.name === disk.name);
+    let size;
+    let storageClass;
+    let other;
```

Comment:
Can't we just `description.push()` directly instead of having intermediate mutable `let` variables?

Reply:
yes! that is much simpler (: I just copied this part